bin/update_lambda_fcn.py

```diff
@@ -131,6 +133,11 @@ def load_lambdas_on_s3(session, domain, bucket):
 
     os.chdir(const.repo_path("lib"))
     zip.write_to_zip('heaviside.git', zipname)
+
+    # Let lambdas look up names by creating a bossnames module.
+    zip.write_to_zip('names.py', zipname, arcname='bossnames/names.py')
```

Comment:
If you don't want to have to reference `bossnames.names.AWSNames` you could probably copy `names.py` as `__init__.py` and then reference as `bossnames.AWSNames`